analytics_data_api/v0/views/learners.py

```diff
@@ -85,55 +90,63 @@ class LearnerListView(CourseViewMixin, generics.ListAPIView):
     **Response Values**
 
         Returns a paginated list of learner metadata and engagement data.
-        Pagination data is returned in the top-level of the returned JSON
-        object:
+        Pagination data is returned in the top level of the returned JSON
+        object.
 
-            * count: The number of learners matching the query.
+            * count: The number of learners that match the query.
             * page: The current one-indexed page number.
             * next: A hyperlink to the next page if one exists, otherwise null.
-            * previous: A hyperlink to the previous page if one exists, otherwise null.
+            * previous: A hyperlink to the previous page if one exists,
+              otherwise null.
 
         The 'results' key in the returned JSON object maps to an array of
-        learners which contains at most a full page's worth of learners.  Each
-        learner is a JSON object containing the following keys:
+        learners that contains, at most, a full page's worth of learners. For
+        each learner there is a JSON object that contains the following keys.
 
             * username: User's username.
-            * enrollment_mode: Enrollment mode (e.g. "honor).
+            * enrollment_mode: Enrollment mode (for example, "verified").
             * name: User's full name.
             * email: User's email.
-            * segments: Classification for this course based on engagement, (e.g. "has_potential").
+            * segments: Classification, based on engagement, of each learner's
+              work in this course (for example, "highly_engaged" or
+              "struggling").
             * engagements: Summary of engagement events for a time span.
-                * videos_viewed: Number of times a video was played.
-                * problems_completed: Unique number of problems completed.
-                * problems_attempted: Unique number of problems attempted.
-                * discussions_contributed: Number of discussions (e.g. forum posts).
+                * videos_viewed: Number of times any course video was played.
+                * problems_completed: Number of unique problems the learner
+                  answered correctly.
+                * problems_attempted: Number of unique problems attempted.
+                  This is a count of the different problems the learner tried.
+                * discussions_contributed: Number of posts, responses, or
+                  comments the learner contributed to course discussions.
```

Comment:
I commented on these descriptions above. I don't have trouble with the wording, but the meaning of the fields isn't what I would have expected.